sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -65,7 +65,6 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
                             // component properties
                             Properties =
                             {
-                                { DigitalTwinsJsonPropertyNames.MetadataModel, componentModelId },
```

Comment:
This was supposed to be removed before, but somehow I came back through all my rebases.